solutions/usaco-615.mdx

```diff
@@ -2,16 +2,53 @@
 id: usaco-615
 source: USACO Bronze 2016 February
 title: Milk Pails
-author: Danh Ta Chi Thanh
+author: Ryan Chou (Python), Danh Ta Chi Thanh (C++)
```

Comment:
i don't think we need to specify languages for authors, since most solutions don't have it
```suggestion
author: Ryan Chou, Danh Ta Chi Thanh
```